platform/v1alpha1/types.go

```diff
@@ -0,0 +1,73 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PlatformOperatorSpec defines the desired state of PlatformOperator.
+type PlatformOperatorSpec struct {
+	// PackageName contains the desired OLM-based Operator package name
+	// that is defined in an existing CatalogSource resource in the cluster.
+	//
+	// This configured package will be managed with the cluster's lifecycle. In
+	// the current implementation, it will be retrieving this name from a list of
+	// supported operators out of the redhat-operators catalog included with
+	// OpenShift.
+	// +kubebuilder:validation:Required
+	PackageName string `json:"packageName"`
+}
+
+// ActiveBundleDeployment references a BundleDeployment resource.
+type ActiveBundleDeployment struct {
```

Comment:
Maybe not something to worry about too much while this API is not GA, but...

Have we landed on a strategy for handling platform operators that declare dependencies? If `po.spec.package.name == "foo"` and `foo` depends on `bar`, there will be two bundle deployments that result from this PO: `foo` and `bar`. Do we care about `bar` in this context?

Reply:
Does the POM component continue to interface with rukpak APIs during phase 1 or would it be updating the Resolution cluster resource with any PO constraints/dependencies/etc.?

Reply:
Not sure we've decided yet.

Reply:
Would any dependencies of a platform operator also themselves be considered platform operators? If so, I would expect the POM to create any of those dependencies as PlatformOperators themselves since they block cluster state.

Reply:
Maybe this is could be an open question on the EP for subsequent phases?